UnitTests/Schemas/IFC4X1/beam-varying-extrusion-paths/src/beam-varying-extrusion-paths.cpp

```diff
@@ -53,17 +51,29 @@ class BeamVaryingExtrusionPaths : public VisualTest {
 	}
 
 	virtual std::string TestName() const { return "beam-varying-extrusion-paths"; }
-	virtual std::string Schema() const { return "IFC4x1"; }
+	virtual std::string Schema() const { return "IFC4X1"; }
+
+	const boost::filesystem::path filename = dataPath("beam-varying-extrusion-paths.ifc");
 
 	std::shared_ptr<oip::EXPRESSModel> express_model = nullptr;
 	buw::ReferenceCounted<oip::IfcImporterT<emt::IFC4X1EntityTypes>> importer = nullptr;
-	buw::ReferenceCounted<oip::IfcGeometryModel> model = buw::makeReferenceCounted<oip::IfcGeometryModel>();
+	buw::ReferenceCounted<oip::IfcModel> model = buw::makeReferenceCounted<oip::IfcModel>();
 };
 
 TEST_F(BeamVaryingExtrusionPaths, AllEntitiesAreRead) {
 	EXPECT_THAT(express_model->entities.size(), Eq(67));
 }
 
+TEST_F(BeamVaryingExtrusionPaths, IFCHasAnEssentialEntity) {
+	auto result = std::find_if(express_model->entities.begin(), express_model->entities.end(), [](auto &pair) -> bool { return pair.second->classname() == "IFCBEAMSTANDARDCASE"; });
```

Comment:
We focus on geometrical entities, like extruded solids, profile defs and similar. Please check for those as well.

Reply:
Resolved: 95071ab